activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Add block support to `ActiveSupport::TimeHelpers#travel_back`
```

Comment:
Oh, I just noticed that there is no dot at the end of the sentence. Coul you add dot for consistency with other entries?

Reply:
Also could you change `ActiveSupport::TimeHelpers` to `ActiveSupport::Testing::TimeHelpers`

Reply:
Done